pypiserver/__init__.py

```diff
@@ -141,6 +141,19 @@ def app_from_config(config: RunConfig) -> Bottle:
     return _app.app
 
 
+def setup_routes_from_config(app: Bottle, config: RunConfig) -> Bottle:
+    """Set up additional routes supplied from the config."""
+    def _setup_health_endpoint(app, config):
```

Comment:
I see the `black` checks are failing here asking for an extra line. 
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/12134172/196450792-0a1b7ea5-54bf-464e-9e1a-58ce051e47c8.png">

You can run the autoformatting on all files using `black .` I think.

Reply:
Thank you, I'm sorry for making mistakes with formatting and typing over and over again 😅
I'm a little unclear on what tests should be done on the local side and how.
I later tried to build `venv`, pip installed `dev.pip`, and executed the `check` related command in `ci.yml` step by step
I hope it won't build fail this time

Reply:
Hey, no worries! I'm happy it helped! :)) yes, that's a good point actually, thanks for pointing this out, I think it'll be a nice improvement to contributor docs in general (#368) 👍 Now all looks green, great job 😃